client/landing/gutenboarding/components/badge/index.tsx

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import classNames from 'classnames';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const Badge: React.FunctionComponent = ( { children, className, ...props } ) => (
```

Comment:
Re-using the same badge for plans grid and design grid:

![image](https://user-images.githubusercontent.com/87168/82444193-88987780-9aab-11ea-9960-6e67fd2df1ac.png)

![image](https://user-images.githubusercontent.com/87168/82444206-8df5c200-9aab-11ea-8f82-0586ae1e351d.png)